Formula/epinio.rb

```diff
@@ -0,0 +1,21 @@
+class Epinio < Formula
+  desc "CLI for Epinio, the Application Development Engine for Kubernetes"
+  homepage "https://epinio.io/"
+  url "https://github.com/epinio/epinio/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "aac002d8a89b0b4a3d4ae3f7e78b4b9ae9cdba903174ecc972a30a31bb2fe940"
+  license "Apache-2.0"
+
+  depends_on "go@1.17" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X github.com/epinio/epinio/internal/version.Version=#{version}")
+  end
+
+  test do
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Thanks for the comment, so is the second test with the `epinio app` command not enough?

If that's the case I will add the required arguments to it and make it fails. Right now it's failing because it's missing some parameters.

Reply:
That seems to be triggering some help text right? In that case no, a better test would be to trigger a scenario a regular user would try.

Reply:
Edited the test using a more concrete command, and checking the error output. :slightly_smiling_face: